### ISSUE TITLE
Build issues since v0.13 on arm

### ISSUE THREAD

**Issue by svenstaro** [messense/if-addrs#41]
On rust 1.80 arm-unknown-linux I get this failure since 0.13:
```
 error[E0277]: the trait bound `i32: From<u32>` is not satisfied
  --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/if-addrs-0.13.0/src/posix_not_mac.rs:75:50
   |
75 |                 tv_usec: timeout.subsec_micros().into(),
   |                                                  ^^^^ the trait `From<u32>` is not implemented for `i32`, which is required by `u32: Into<_>`
   |
   = help: the following other types implement trait `From<T>`:
             <i32 as From<bool>>
             <i32 as From<i16>>
             <i32 as From<i8>>
             <i32 as From<u16>>
             <i32 as From<u8>>
   = note: required for `u32` to implement `Into<i32>`
```

**Comment by svenstaro** [comment]
Build log here: https://github.com/svenstaro/miniserve/actions/runs/9778254160/job/26994671116?pr=1439

**Comment by wez** [comment]
Also experiencing this, had to revert.
https://github.com/wez/govee2mqtt/commit/841426dcb47ca3662d40d2d94a477d3df8e1cca0

**Comment by JanKoudijs** [comment]
Looks like this got fixed by #40, but it did not make it into a release yet.
Referring directly to the commit hash fixed the build for me.
Guess we have to wait.

**Comment by messense** [comment]
Fix released in v0.13.1.
